.github/recipe.yaml

```diff
@@ -1,26 +1,28 @@
 plugins:
-  audioplayers: ["wearable-5.5", "tv-7.0"]
-  battery_plus: ["wearable-5.5"]
-  connectivity_plus: ["wearable-5.5", "tv-7.0"]
-  device_info_plus: ["wearable-5.5", "tv-7.0"]
-  flutter_app_badger: ["wearable-5.5"]
-  flutter_secure_storage: ["wearable-5.5", "tv-7.0"]
-  flutter_tts: ["wearable-5.5", "tv-7.0"]
-  integration_test: ["wearable-5.5", "tv-7.0"]
-  messageport: ["wearable-5.5", "tv-7.0"]
-  package_info_plus: ["wearable-5.5", "tv-7.0"]
-  path_provider: ["wearable-5.5", "tv-7.0"]
-  permission_handler: ["wearable-5.5"]
-  sensors_plus: ["wearable-5.5"]
-  shared_preferences: ["wearable-5.5", "tv-7.0"]
-  sqflite: ["wearable-5.5", "tv-7.0"]
-  tizen_app_manager: ["wearable-5.5", "tv-7.0"]
-  tizen_audio_manager: ["wearable-5.5", "tv-7.0"]
-  tizen_bundle: ["wearable-5.5", "tv-7.0"]
-  tizen_package_manager: ["wearable-5.5", "tv-7.0"]
+  audioplayers: ["tv-7.0"]
```

Comment:
Note that the latest TV emulator version is 8.0. You can upgrade the package installed on our integration test server (self-hosted runner).

Reply:
Is there a reason(history?) you used 7.0? Is it just because it was the latest version when you set it up?

Reply:
We can add both the tizen 8.0 emulator and the 8.0 tv emulator. I will make another PR.
I think the first thing to do is to fix the build fail.

Reply:
> Is it just because it was the latest version when you set it up?

Yes, it was originally 6.0 and updated last year.

Reply:
Oh my mistake, I already updated the TV emulator version 8.0 on the host PC. I will update this PR and add tizen emulator in a new PR.